### PR TITLE
Module for managing global Parameter within Foreman

### PR DESCRIPTION
**Pull request by Fobhep**

### CODE REVIEW

**Comment by ehelms** [comment]
@Fobhep Thanks for the module -- would you mind adjusting this PR so its just your single commit?

**Comment by Fobhep** [comment]
@ehelms yes! I am on that- sorry for the mess I am creating here!

**Comment by ehelms** [comment]
@Fobhep If I could help, I'd recommend resetting your local branch to origin/master and then cherry-picking your fix on top of it and pushing that backup to the PR.

**Comment by Fobhep** [comment]
;) just did that

**Comment by ehelms** [comment]
@Fobhep Want to add your module to the README as well?

**Comment by Fobhep** [comment]
ah yes, of course. I will ammend that!

**Comment by Fobhep** [comment]
I made a new commit now, not an amendment. hope that is fine.

File: modules/foreman_global_parameter.py [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+
+- name: "Delete a Global Parameter"
+  local_action:
+      module: foreman_global_parameter
+      username: "admin"
+      password: "admin"
+      server_url: "https://fakeserver.com"
+      name: "TheAnswer"
+      state: absent
+'''
```
Comment (ehelms) [inline modules/foreman_global_parameter.py]: Do you mind updating the examples to follow the updated way of declaring them such as https://github.com/theforeman/foreman-ansible-modules/blob/master/modules/katello_publish.py#L54 ? This reduces amount of declared things and makes the module top level.

**Comment by Fobhep** [comment]
I updated the example block now. Did you mean it that way?

**Comment by ehelms** [comment]
Yup! Looks good @Fobhep, thanks !

**Comment by ehelms** [comment]
Ha, so @Fobhep I just realized your PR was against a different branch than master. I will cherry pick this and push it into master to clear things up.

**Comment by Fobhep** [comment]
I did not feel comfortable to put a pull request against your master branch :)
add_modules seemed a more appropriate choice. However thanks for accepting it ;)

**Pull request merged**